index.html

```diff
@@ -808,13 +840,23 @@ <h2>
           </li>
         </ol>
       </section>
-      <section>
+      <section id="check-permission">
         <h2>
           Check permission
         </h2>
         <p>
           The <cite>Geolocation API</cite> is a [=default powerful feature=].
         </p>
+        <p>
+          It is RECOMMENDED that a user agent prioritize restricting the
+          [=permission=] [=permission/lifetime=] to a single session: This can
```

Comment:
This is stricter than the examples above, which start with time-based lifetimes. Could you make them consistent?